prusti-viper/src/encoder/middle/core_proof/builtin_methods/interface.rs

```diff
@@ -291,6 +294,26 @@ impl<'p, 'v: 'p, 'tcx: 'v> Private for Lowerer<'p, 'v, 'tcx> {
     ) -> SpannedEncodingResult<()> {
         use vir_low::macros::*;
         let assigned_value = match value {
+            vir_mid::Rvalue::Ref(value) => {
+                let ty = value.place.get_type();
+                var_decls! {
+                    operand_place: Place,
+                    operand_address: Address,
+                    operand_value: { ty.create_snapshot(self)? }
+                };
+                let predicate = expr! {
+                    acc(OwnedNonAliased<ty>(operand_place, operand_address, operand_value))
+                };
+                let compute_address = ty!(Address);
+                let address =
+                    expr! { ComputeAddress::compute_address(operand_place, operand_address) };
+                pres.push(predicate.clone());
+                posts.push(predicate);
```

Comment:
Shouldn't this push `MutBorrowedNonAliased`/`SharedBorrowedNonAliased` instead?

Reply:
yes. added them.